tools/DependencyUpdater/Program.cs

```diff
@@ -72,25 +72,15 @@ internal class Program
             }
 
             // check if this is running on a git repo
-            string gitRepo = "";
-            if (!RunGitCli("remote -v", ref gitRepo, workingDirectory))
-            {
-                Environment.Exit(1);
-            }
-
-            if (gitRepo.Contains("fatal: not a git repository"))
-            {
-                Console.WriteLine($"ERROR: working directory is not a git repository");
-                Environment.Exit(1);
-            }
+            string gitRepo = GetGitRepoFromWorkingDirectory(workingDirectory);
```

Comment:
Changed to ```CheckIfDirectoryIsGitRepo``` + added summary to method

But, this method in my opinion should be changed. It should not exit application, rather return object with two properties: bool IsValidRepo and string RepositoryAddress. Then based on flag we should either:
- Exit application in main loop
- In "multi repository path" - write error to console output and progress with others (tool can print a summary after loop)
Let me know what you think about it.

Reply:
I see. Valid points you make. OK with exiting only on main() and OK with not exiting on multi repo workflow and instead output a message and continue to the next one.

Reply:
I've change foreach update look, now it's iterating through each lib and then prints output. Also, if there were any errors it will exit with code "1".